template.yaml

```diff
@@ -45,6 +45,11 @@ Resources:
   NvaCristinProjectsFunctionApi:
     Type: AWS::Serverless::Api
     Properties:
+      DefinitionBody:
+        'Fn::Transform':
+          Name: AWS::Include
+          Parameters:
+            Location: ./docs/projects-swagger.yaml
```

Comment:
Has this been tested? Do you need to include a S3 bucket location?

Reply:
As far as I know, I have no way to test a template. This code was an example given by @svenngjeto so I just have to assume it works. If you have some suggestions please let me know

Reply:
Verified in CFN deploy.